src/empty/index.ts

```diff
@@ -0,0 +1,7 @@
+import { Store } from 'effector';
+
+import { equals } from '../equals';
+
+export function empty<A>(a: Store<A | null>): Store<boolean> {
+  return equals(a, null);
```

Comment:
I propose not using methods inside other methods.
It possibly can break SSR, because required to wrap this line with `withFactory` inside patronum's code.

```suggestion
  return source.map(value => value === null);
```

Reply:
At the moment I don't know how it will work with separate imports and tree-shaking:

```ts
import { empty } from 'patronum/empty';
// Will `equals` be imported as 'patronum/equals' or be bundled inside `empty`?
```